gloss/sys_sbrk.c

```diff
@@ -8,26 +8,31 @@
  * that's been allocated. */
 extern char metal_segment_heap_target_start;
 extern char metal_segment_heap_target_end;
-static char *brk = &metal_segment_heap_target_start;
+static char *__brk = &metal_segment_heap_target_start;
+
+#ifdef _PICOLIBC__
+#define _brk brk
+#define _sbrk sbrk
+#endif
 
 int _brk(void *addr) {
-    brk = addr;
```

Comment:
what set brk now? is it newlib? how come there was no redefinition error previously?

Reply:
I had to rename the local 'brk' variable because picolibc uses that for the function newlib calls '_brk'. If you read the POSIX spec, 'brk' is the usual name; I'm not sure why newlib changed it.